shaptools/hana.py

```diff
@@ -448,3 +448,135 @@ def get_sr_status(self):
         # (see SAPHana RA)
         status["status"] = SR_STATUS.get(result.returncode, SR_STATUS[12])
         return status
+
+    def _manage_ini_file(
+            self, parameter_str, database, file_name, layer,
+            **kwargs):
+        """
+        Construct command with HANA SQL to update configuration parameters in ini file
+
+        key_name or user_name/user_password parameters must be used
+        Args:
+            parameter_str(list): list containing HANA parameter details in a dict format
```

Comment:
space missing after `parameter_str` and param description should start in capital letter.

Please, check this kind of things in the rest of the code